index.html

```diff
@@ -191,49 +195,44 @@ <h2 class ="supporting--text">LET’S CONNECT</h2>
             </ul>  
      </div>
      <button class="resume">Get my resume</button>
-     
+     </div>
      <div class = "abt-languages">
+     <div class="language-text">
         <p>Languages</p>
         <img src="./images/dropdown.png" alt="">
      </div>
      <div class="prog--lang">
         <ul>
-            <li>
-                <div class = "lang--container">
-                    <img class="lang-logo" src="./images/javascript.png" alt="javascript logo">
-                    <p class="lang--name">Javascript</p>
-                </div>
-            </li>
-            <li>
-                <div class = "lang--container">
-                    <img class="lang-logo" src="./images/html.png" alt="html image">
-                    <p class="lang--name">HTML</p>
+            <li class="lang-lists">
+              <div class = "lang--container">
+                <img class="lang-logo" src="./images/javascript.png" alt="javascript logo">
+                <p class="lang--name">Javascript</p>
+              </div>
+              <div class = "lang--container">
+                <img class="lang-logo" src="./images/html.png" alt="html image">
+                <p class="lang--name">HTML</p>
+              </div>
+              <div class = "lang--container">
+                <img class="lang-logo" src="./images/css.png" alt="css image">
+                <p class="lang--name">CSS</p>
```

Comment:
- [ ] For semantic purposes, Kindly consider grabbing your tech stack inside a list tag element.
`ul, li`